src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1028,21 +1040,66 @@ public void setConvertToUpperCase(boolean convertToUpperCase) {
          */
         @Override
         public Set getGroupMembershipRoles(String userDn, String username) {
-            Set<GrantedAuthority> names = super.getGroupMembershipRoles(userDn,username);
+            if (ldapSearch == null) {
+                LOGGER.log(Level.WARNING, "[JENKINS-38124] Please update $JENKINS_HOME/LDAPBindSecurityRealm.groovy file to fix it.");
```

Comment:
Create a sub-page on the LDAP plugin wiki page with details of how somebody is supposed to fix this issue and include a link to that sub-page